django/contrib/postgres/functions.py

```diff
@@ -1,4 +1,13 @@
-from django.db.models import DateTimeField, Func
+from django.db.models import DateTimeField, Func, UUIDField
+
+
+class RandomUUID(Func):
+    template = 'GEN_RANDOM_UUID()'
+
+    def __init__(self, output_field=None, **extra):
+        if output_field is None:
+            output_field = UUIDField()
```

Comment:
I think you could use `_output_field = UUIDField()` at the class level instead of this?